http-client/src/test/java/io/airlift/http/client/jetty/AbstractHttpClientTestHttpProxy.java

```diff
@@ -51,7 +53,7 @@ public void setUpHttpClient()
             throws Exception
     {
         testingHttpProxy = new TestingHttpProxy(Optional.ofNullable(keystore));
-        httpClient = new JettyHttpClient("test-shared", createClientConfig(), ImmutableList.of(new TestingRequestFilter()));
+        httpClient = new JettyHttpClient("test-shared", createClientConfig(), ImmutableList.of(new TestingRequestFilter()), ImmutableSet.of(new TestingStatusListener(statusCounts)));
```

Comment:
Maybe add a method `getStatusListeners()` in the parent class.

Reply:
We don't have it for filters and it's not needed anywhere